docs/DeveloperGuide.md

```diff
@@ -286,10 +286,38 @@ In `MainWindow#executeCommand(String commandText)`, the type of `CommandResult`
     * isHelpCommand - boolean indicating whether the command is related to getting help
     * isExitCommand - boolean indicating whether the command is to exit the application
 
-
+    
 The following activity diagram summarizes what happens when a user executes the different types of commands:
 ![DataTogglingActivityDiagram](images/DataTogglingActivityDiagram.png)
```

Comment:
Something missing?

Reply:
Seems like an extra tab space; I've removed it from my commits